src/PrestaShopBundle/Resources/config/routing/admin/sell/orders/orders.yml

```diff
@@ -79,3 +79,21 @@ admin_orders_add_payment:
     _controller: PrestaShopBundle:Admin/Sell/Order/Order:addPayment
   requirements:
     orderId: \d+
+
+admin_orders_update_product:
+  path: /{orderId}/products/{orderDetailId}
+  methods: [POST]
+  defaults:
+    _controller: PrestaShopBundle:Admin/Sell/Order/Order:updateProduct
```

Comment:
Legacy links ?
Maybe at the end of the order page project ?

Reply:
Yup, at the end of page migration, otherwise it will enable this page too early.